### PR TITLE
Pending Trips Frontend Part 1: Trip View Tabs

### PR DESCRIPTION
**Pull request by zghera**
### What is a quick description of the change?

Create the manage trips bar which includes the trips view nav. This nav allows the user to select the trip view ([pending trip invites feature](https://docs.google.com/document/d/14ne4ZLJrYMpaX-LS08rYOD6a2r6A3NTZV1tmfb_AcTk/edit#bookmark=id.fffbvo5j9dvp)).

### Is this fixing an issue?

None.

### Are there more details that are relevant?

None.

### Check lists (check `x` in `[ ]` of list items)

- [  ] Test written/updating
- [  ] Tests passing
- [x] Coding style (indentation, etc)

Only UI based changes.

Same manual tests as in #123 but ran into issue #134.

### Any additional comments?

Before:
icons-styling

After:
After-pending-trip-frontend

### CODE REVIEW

File: frontend/src/components/ViewTrips/manage-trips-bar.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+          defaultActiveKey={active}
+        >
+          <Nav.Item>
+            <Nav.Link
+              eventKey={active}
+              onSelect={() => props.handleChangeView(active)}
+            >
+              Active
+            </Nav.Link>
+          </Nav.Item>
```
Comment (ananyeet) [inline frontend/src/components/ViewTrips/manage-trips-bar.js:38]: Make a function to generate each of these? Up to you.

File: frontend/src/components/ViewTrips/trips-container.js [hunk 2]
```diff
@@ -65,7 +65,7 @@ class TripsContainer extends React.Component {
    *
    * @override
    */
-  async componentDidMount() {
+  async componentDidUpdate() {
```
Comment (ananyeet) [inline frontend/src/components/ViewTrips/trips-container.js:68]: This looks like a change from a different PR?

Comment (zghera) [reply frontend/src/components/ViewTrips/trips-container.js:68]: It is not. This was needed in order to update the TripsContainer whenever the trip view (change tabs) is updated. This also caused #134 but that is fixed in the following PR.

File: frontend/src/components/ViewTrips/manage-trips-bar.js (hunk 1 above)
Comment (zghera) [reply frontend/src/components/ViewTrips/manage-trips-bar.js:38]: Yeah that would look a lot cleaner, I will add that. Thanks for the suggestion!

**Comment by zghera** [comment]
Received @Hiram-Silvey 's verbal approval to merge since this is the last day of the internship and he was not able to review the PR.

**Pull request merged**